app/Sidebar/Sidebar.php

```diff
@@ -11,72 +11,108 @@ public function register(): void
         add_action('render_block_core/navigation-link', [$this, 'addIconsToBlockMenuItems'], 10, 2);
     }
 
-    /**
-     * Adds icons to block menu items based on ACF field values.
-     */
     public function addIconsToBlockMenuItems($blockContent, $block): string
     {
         if ('core/navigation-link' !== $block['blockName']) {
```

Comment:
Dit is niet nodig aangezien je al de block-specifieke action gebruikt.

Reply:
ja maar ja maar, safety 👀